build.gradle

```diff
@@ -0,0 +1,118 @@
+buildscript {
```

Comment:
We won't be changing to gradle (see #18)

Reply:
It's not about changing to Gradle, I added it in addition because it makes building the project easier for me. But I can take it out of this branch.

Reply:
I can understand that. Unfortunately, when we add it, even as an addition, it means we need to maintain it. Feel free to keep it in for yourself, without committing it in this branch.